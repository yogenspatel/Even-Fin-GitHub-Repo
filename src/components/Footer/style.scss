footer {
    background-color: #4388c4;
    height: 48px;
    position: fixed;
    width: 100%;
    bottom: 0;
}

footer p {
    margin: auto;
    width: 325px;
    color: #fff;
    padding: 10px 0 0 0;
}
