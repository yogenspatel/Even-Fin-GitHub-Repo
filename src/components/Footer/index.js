import React from 'react';
import './style.scss';

const Footer = () => (
  <footer>
    <p>&copy; {new Date().getFullYear()} Yogen Patel</p>
  </footer>
);

export default Footer;
