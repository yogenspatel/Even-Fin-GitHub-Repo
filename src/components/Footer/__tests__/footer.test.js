/* eslint-disable no-undef */

import React from "react";
import ReactDOM from "react-dom";
import Footer from "../../Footer";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

it("Footer renders without crashing", () => {
  const footer = document.createElement("footer");
  ReactDOM.render(<Footer />, footer);
});

it("Footer Match Snapshot", () => {
  const wrapper = shallow(<Footer />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
