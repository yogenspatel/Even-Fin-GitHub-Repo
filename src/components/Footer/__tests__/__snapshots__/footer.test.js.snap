// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Match Snapshot 1`] = `
<footer>
  <p>
    Â© 
    2019
     Even Financial, Inc. - CONFIDENTIAL
  </p>
</footer>
`;
